src/main/java/com/exasol/errorcodecrawlermavenplugin/ErrorMessageDeclarationCrawler.java

```diff
@@ -85,6 +92,10 @@ private SpoonAPI initSpoon(final Path... pathsToCrawl) {
      */
     private void crawl(final CtInvocation<?> methodInvocation, final List<Finding> findings,
             final List<ErrorMessageDeclaration> errorMessageDeclarations) {
+        final File sourceFile = methodInvocation.getPosition().getFile();
```

Comment:
Please also consider removing the `List<Finding> findings` and put it as return type.

Reply:
I prefer it like this since I would have to decide if I should return the findings or the errorMessageDeclarations. By doing both using a parameter it's at least consistent.